installer/scripts/tue-install-impl.bash

```diff
@@ -467,6 +468,20 @@ function tue-install-snap
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+function tue-install-dpkg
```

Comment:
Are we using it anywhere? Not sure if any of the targets have a dpkg command in the install scripts

Reply:
Nope, not using it (yet)